application/templates/application/application-overview.html

```diff
@@ -0,0 +1,140 @@
+{% block app_overview %}
+<div style="height: 90%; margin-bottom: 5%;">
+    {% if selected_app %}
+    <div class="top-header">
+        <div class="right-pane-container">
+            {% if not selected_app.is_draft %}
+            <h5 style="margin: 1em; text-align: center;">
+                <b>Application Form</b>
+            </h5>
+            <table class="table table-striped" style="border: 1px solid black; border-collapse:
+                inherit; width: 75%; margin: 0 auto 2em;">
+                <tbody>
+                    {% for name, value in selected_app.get_fields %}
+                    <tr>
+                        <th scope="row">{{name}}</th>
+                        <td>{{ value }}</td>
+                    </tr>
+                    {% endfor %}
+                </tbody>
+            </table>
+            {% else %}
+            <form action="{% url 'dashboard:application:draftExistingApplication' selected_app.pk %}"
+                method="post" class="form-signup">
+                {% csrf_token %}
+                <div class="form-row">
+                    <div class="form-label-group col-md-6">
+                        {{ form.first_name }}
```

Comment:
The field and label orientation is different in this file and "application-form.html" (field comes before label here and the other way round in application-form.html). Please adhere to a common design strategy (maybe refer to the login/index.html for better clarity). Please comment if there's a specific reason behind this choice.

Reply:
the content in this file is correct, i will change it in application-form.html